JS/localStorage.js

```diff
@@ -0,0 +1,172 @@
+          <ul>
+          ${technologies.map((technology) => `<li>${technology}</li>`).join('')}
+          </ul>
+      </div>
+  </div>
+  <button data-modal-target="#modal" class="btn btn-style btn-style-btn " onclick="showModal('${id}')" id="" href="${linkVersion}" target="_blank">${linkVersion}</button>
+</div>
+
+  `;
+
+  return container;
+};
+
+const renderPopupps = () => {
+  const container = document.querySelector('.container-block');
+
+  const popuppsElement = document.createElement('div');
+  const containerFull = document.createElement('div');
+  containerFull.classList.add('container-full');
+  popuppsElement.appendChild(containerFull);
+  popuppsElement.classList.add('container-form');
+
+  popupps.forEach((popupp) => {
+    containerFull.appendChild(renderPopupp(popupp));
+    container.appendChild(popuppsElement);
+  });
+};
+renderPopupps();
+
+const showModal = (id, isMultipost) => {
+  const modal = document.getElementById('myModal');
+  modal.style.display = 'block';
+  if (isMultipost) {
+    document.getElementById('model-dynamic-content').innerHTML = `
+    <div class='pupopp-item'>
+    <img src="./imgs/Img_Placeholder.svg" alt="here's placeholder image where the alternatives text goes">
+  <div class="section-art sm-screen">
+      <h2>Multi-Post Stories</h2>
+      <p>
+      A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+      </p>
+      <div class="luang">
+          <ul>
+          <li>CSS</li>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+          </ul>
+      </div>
+      <a data-modal-target="#modal" class="btn btn-style btn-style-btn " href="" target="_blank">See Live</a>
+      <a data-modal-target="#modal" class="btn btn-style btn-style-btn " href="" target="_blank">See Source</a>
+  </div>
+  </div>
+    `;
+  } else {
+    const item = popupps.find((p) => p.id.toString() === id);
+    const {
+      name, description, featuredImage, technologies, linkVersion,
+    } = item;
+
+    document.getElementById('model-dynamic-content').innerHTML = `
+  <div class='pupopp-item'>
+  <div data-id=${id} class='pupopp lg-screen pupopp-form${id}' style="background:url(${featuredImage}); width: 97vh;"></div>
+  <div class="section-art sm-screen">
+      <h2>${name}</h2>
+      <p>
+      ${description}
+      </p>
+      <div class="luang">
+          <ul>
+          ${technologies.map((technology) => `<li>${technology}</li>`).join('')}
+          </ul>
+      </div>
+      <a data-modal-target="#modal" class="btn btn-style btn-style-btn " onclick="showModal('${id}')" id="" href="${linkVersion}" target="_blank">${linkVersion}</a>
+      <a data-modal-target="#modal" class="btn btn-style btn-style-btn " onclick="showModal('${id}')" id="" href="${linkVersion}" target="_blank">${linkVersion}</a>
+  </div>
+  </div>
+  `;
+  }
+};
```

Comment:
- [ ] Good job so far your popup is displaying :+1: But it is not working as expected :cry:  For Desktop your popup does not conform to the dimensions set in the Figma template. You can see a difference when you compare it to Figma. Please use the dimensions and follow the layout as defined in the Figma template to make them similar. :smile: 

### Figma template
![Screenshot from 2022-07-19 19-31-42](https://user-images.githubusercontent.com/6615394/179837220-be1e4a1c-4449-4f77-bccd-e6be8dc2abff.png)

### Your project
![Screenshot from 2022-07-19 19-31-13](https://user-images.githubusercontent.com/6615394/179837229-4eb01122-5443-4dd4-96da-61efec4d6a70.png)




- [ ] For the Mobile version the details about the project do not appear. The elements are improperly placed as you can see in the picture below :disappointed: Please fix it so they can appear on the popup so it looks similar to the Figma template. :+1: 

### Your Mobile popup
![Screenshot from 2022-07-19 19-32-53](https://user-images.githubusercontent.com/6615394/179838258-3ab660a1-bd95-4cbb-b890-b7fdf794ea72.png)

### Figma mobile popup
![Screenshot from 2022-07-19 19-32-11](https://user-images.githubusercontent.com/6615394/179838263-04791119-846a-4d81-99a6-496ed91d626f.png)